SwordImportExportPlugin.inc.php

```diff
@@ -27,6 +29,7 @@ class SwordImportExportPlugin extends ImportExportPlugin {
 	public function __construct($parentPlugin) {
 		parent::__construct();
 		$this->_parentPlugin = $parentPlugin;
+		$this->_version = Application::getApplication()->getCurrentVersion();
```

Comment:
`Application::get()` is preferred (I'll add a deprecation note to `getApplication`).